packages/editor/src/utils/firebaseUtils.tsx

```diff
@@ -0,0 +1,163 @@
+import { initializeApp } from "firebase/app";
+import {
+  GithubAuthProvider,
+  getAuth,
+  signInWithPopup,
+  signOut,
+} from "firebase/auth";
+import {
+  collection,
+  doc,
+  getDoc,
+  getDocs,
+  getFirestore,
+} from "firebase/firestore";
+import toast from "react-hot-toast";
+import { useRecoilCallback } from "recoil";
+import { SavedWorkspaces, Workspace, settingsState } from "../state/atoms.js";
+
+const firebaseConfig = {
```

Comment:
we discussed offine: it might be good practice to put this in the CI/CD environment instead.